core/commands/files.go

```diff
@@ -735,6 +736,7 @@ stat' on the file or any of its ancestors.
 		}
 
 		create, _ := req.Options["create"].(bool)
+		dashp, _ := req.Options["parents"].(bool)
```

Comment:
Let's call this "mkParents" or something like that. Contrary to what the rest of the go-ipfs code might indicate, we don't pay by the letter in variable names :smile:.